src/mudlet.pro

```diff
@@ -1339,4 +1331,8 @@ DISTFILES += \
     mudlet-lua/tests/README.md \
     mudlet-lua/tests/DB.lua \
     mudlet-lua/tests/GUIUtils.lua \
-    mudlet-lua/tests/Other.lua
+    mudlet-lua/tests/Other.lua \
+    ../mudlet.desktop \
+    ../mudlet.png \
+    ../mudlet.svg \
+    ../README.md
```

Comment:
I have revised this file slightly for FreeBSD and having it listed in Qt Creator seems sensible to me.

The two image files are likely to be referenced by the `mudlet.desktop` file (for use as the application icon on FreeDesktop compliant/using systems - currently FreeBSD and Linux) so it makes sense to have them to hand in Qt Creator.